examples/markdown-color.ftd

```diff
@@ -0,0 +1,13 @@
+-- boolean flag: true
+
+
+-- ftd.link-code:
+if: $flag
+light: purple
+dark: orange
```

Comment:
Lets create a single `ftd.markdown-colors` variable of type `ftd.markdown-color-data`, so we have a single ftd variable to document.